src/git/install.sh

```diff
@@ -16,16 +16,51 @@ keyserver hkp://keyserver.ubuntu.com:80
 keyserver hkps://keys.openpgp.org
 keyserver hkp://keyserver.pgp.com"
 
-set -e
-
-# Clean up
-rm -rf /var/lib/apt/lists/*
-
 if [ "$(id -u)" -ne 0 ]; then
     echo -e 'Script must be run as root. Use sudo, su, or add "USER root" to your Dockerfile before running this script.'
     exit 1
 fi
 
+# Bring in ID, ID_LIKE, VERSION_ID, VERSION_CODENAME
+. /etc/os-release
+# Get an adjusted ID independent of distro variants
+if [ "${ID}" = "debian" ] || [ "${ID_LIKE}" = "debian" ]; then
```

Comment:
```suggestion
if [ "${ID}" = "debian" ] || [ "${ID_LIKE}" = "ubuntu" ]; then
```

Reply:
Looks like `common-utils` Feature need a similar change, but we can isolate and make changes in a separate PR. See https://github.com/devcontainers/features/blob/3e96db3a6a28667bdc012ebecc3c798ddad6af99/src/common-utils/main.sh#L335

Reply:
This check is correct and does not need a PR. 

For Ubuntu images `ID=ubunto` and `ID_LIKE=debian`. 
For Debian images `ID_LIKE` is not set.

The purpose here is to recognize that Ubuntu should be treated like debian in the script, same for `common-utils`.

Jeff.

Reply:
Hah! Apologies, I definitely misread the code line.